opal/mca/common/ofi/common_ofi.c

```diff
@@ -40,6 +40,10 @@ static int opal_common_ofi_init_ref_cnt = 0;
 
 #if OPAL_OFI_IMPORT_MONITOR_SUPPORT
 
+/*
+ * These no-op functions are necessary since libfabric does not allow null
+ * function pointers here or else it will segfault.
```

Comment:
drop the "or else it will segfault".  The fact that it doesn't allow NULL pointers is sufficient.